my-dsl.md

```diff
@@ -1,38 +1,32 @@
 # Language
-_What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+[discord.py](https://github.com/Rapptz/discord.py)
 
 
 # Domain
-_Describe the language's domain in five words._
+Creating bot clients for Discord
 
 
 # Computational model
-_We don't yet have a great definition of the term "computational model". 
-For now, try to come up with the clearest, most concise explanation of 
-what happens when a program in your DSL runs._
+The Python interpreter executes the program as it would Python code.
 
 
 # DSL-ness
-_Fowler writes about a spectrum of languages, from general-purpose languages to 
-"purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
-and why?_ 
+Extremely domain specific. This cannot be used to do anything other than creating Discord bots.
```

Comment:
I agree that discord.py is focused on one domain, but it looks very API-like and still feels connected to the host language (Python). Also, Fowler distinguishes APIs, which have vocabulary, and internal DSLs, which have grammar and make sense when vocabulary "words" are strung together (page 29). Looking at examples on GitHub, most of the discord.py usage is like `client.singleMethod()` as opposed to a string of methods. So discord.py looks more like an API than a DSL (according to Fowler).